controllers/email-l10n.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const HBSHelpers = require("../hbs-helpers");
+
+
+function testEmails(req, res) {
+  const unsafeBreachesForEmail = [];
+  const email = "example@email.com";
+  let breachAlert;
+  let buttonValue = req.fluentFormat("verify-my-email");
+
+  if (!req.query.view) {
+    req.query.view = "email_verify";
+  }
+
+  if (req.query.type) {
+    buttonValue = req.fluentFormat("report-scan-another-email");
+    if(req.query.type === "singleBreach" || req.query.type === "breachAlert") {
+      unsafeBreachesForEmail.push(req.app.locals.breaches.filter(breach => breach.Name === "Experian")[0]);
+      if(req.query.type === "breachAlert") {
+        breachAlert = unsafeBreachesForEmail[0];
+      }
+    } else if (req.query.type === "multipleBreaches") {
+      const breachArray = ["Experian", "Dropbox", "Apollo"];
+      breachArray.forEach(name => {
+        unsafeBreachesForEmail.push(req.app.locals.breaches.filter(breach => breach.Name === name)[0]);
+      });
+    }
+  }
+
+  res.render("test_emails", {
+    layout: "email_l10n_testing.hbs",
+    unsafeBreachesForEmail: unsafeBreachesForEmail,
+    supportedLocales: req.supportedLocales,
+    whichView: `email_partials/${req.query.view}`,
```

Comment:
Does this need some validation?
I see we're guarding against a null/missing `req.query.view` on L12-14 above, but not sure what would happen if the query `?view=` is somehow malformed or something. Although looking at this again, I see its a `testEmail()` method and "test_emails" view, so maybe this is just debugging code.

Reply:
No debugging here,  just totally crummy nomenclature on my part! Changing this to something less completely misleading and will add something to catch query malformations.